lib/errors.js

```diff
@@ -19,4 +19,28 @@ module.exports = class CryptoError extends Error {
       CryptoError.UNSUPPORTED_DIGEST_METHOD
     )
   }
+
+  static UNSUPPORTED_ALGORITHM(msg) {
+    return new CryptoError(
+      msg,
+      'UNSUPPORTED_ALGORITHM',
+      CryptoError.UNSUPPORTED_DIGEST_METHOD
```

Comment:
```suggestion
      CryptoError.UNSUPPORTED_ALGORITHM
```

Same for the ones below.